src/graph_builder.js

```diff
@@ -0,0 +1,27 @@
+import Graph from './rdf/graph'
+import Statement from './rdf/statement'
+/**
+ * Builds RDF graphs from the Redux state
+ */
+export default class GraphBuilder {
+  /**
+   * @param {Object} state the Redux state
+   */
+  constructor(state) {
+    console.log(state)
+    this.state = state
+  }
+
+  /**
+   * @return {Graph} an RDF graph that represents the data in the state
+   */
+  get graph() {
+    const graph = new Graph()
+    for(var key in this.state) {
```

Comment:
Prefer `const` to `var`. (Will be adding some eslint coverage of this.)

Reply:
Doesn't it get reassigned each time through the loop?

Reply:
Er, yes it does. Sorry, meant `let`!